src/constants/index.ts

```diff
@@ -20,3 +20,5 @@ export const POKEMON_TYPE_COLORS = {
 } as const;
 
 export type PokemonTypes = keyof typeof POKEMON_TYPE_COLORS;
+
+export const SEARCH_HEIGHT = 70;
```

Comment:
Personally I prefer to define such constants close to place that they are used. It is easier to understand why they are there. And to updated/remove them during refactors